icu4c/source/common/cjmlbe.h

```diff
@@ -0,0 +1,170 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+#ifndef CJMLBREAKENGINE_H
+#define CJMLBREAKENGINE_H
+
+#include "unicode/utypes.h"
+#include "unicode/uniset.h"
+#include "unicode/utext.h"
+
+#include "brkeng.h"
+#include "dictbe.h"
+#include "hash.h"
+#include "uvectr32.h"
+
+#include <queue>
```

Comment:
ICU is restricted to a small subset of the C++ STL. In particular, we cannot use anything that allocates memory or throws exceptions. We use UVector, UVector32, UVector64, and home-grown lists.

Reply:
Done. Remove it.